test/jdk/java/awt/ScrollPane/ScrollPaneRemoveAdd.java

```diff
@@ -76,22 +76,23 @@ public void actionPerformed(ActionEvent e) {
     public void start() throws Exception {
         try {
             EventQueue.invokeAndWait(() -> {
-                    frame.pack();
-                    frame.setLocationRelativeTo(null);
-                    frame.setVisible(true);
-                    pane.remove(0);
-                    pane.add(button);
+                frame.pack();
+                frame.setLocationRelativeTo(null);
+                frame.setVisible(true);
+                pane.remove(0);
+                pane.add(button);
             });
 
             Point buttonLoc = button.getLocationOnScreen();
             Dimension buttonSize = button.getSize();
-            robot.mouseMove(buttonLoc.x + buttonSize.width / 2, buttonLoc.y + buttonSize.height / 2);
+            robot.mouseMove(buttonLoc.x + buttonSize.width / 2,
+                    buttonLoc.y + buttonSize.height / 2);
             robot.delay(50);
             robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
             robot.delay(50);
             robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
 
-            Toolkit.getDefaultToolkit().sync();
+   //         Toolkit.getDefaultToolkit().sync();
```

Comment:
Instead of commenting out the line, remove it.

Reply:
Updated.